components/molecules/Greeting.js

```diff
@@ -5,10 +5,25 @@ import fr from '../../locales/fr'
 export default function Greeting(props) {
   const t = props.locale === 'en' ? en : fr
 
+  function getGreeting() {
+    switch (true) {
+      case props.time < 12:
+        return t.morningGreeting
+      case props.time < 17:
+        return t.afternoonGreeting
+      case props.time < 24:
+        return t.eveningGreeting
+      default:
+        return t.greeting
```

Comment:
The logic seems sound, except this is probably best suited for an if statement then a switch.
As well, with AEM, this component won't have access to the `t` translation. This would be a good time to remove it altogether from this file and have the dashboard page call a lib function to pass the right text directly to the component. Should be very similar to how the UniversalBenefitCard component works.